plugins/source/mixpanel/resources/services/export/export_events.go

```diff
@@ -0,0 +1,63 @@
+package export
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/cloudquery/cloudquery/plugins/source/mixpanel/client"
+	"github.com/cloudquery/cloudquery/plugins/source/mixpanel/internal/mixpanel"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+	"github.com/pkg/errors"
+)
+
+func ExportEvents() *schema.Table {
+	return &schema.Table{
+		Name:      "mixpanel_export_events",
+		Resolver:  fetchExportEvents,
+		Transform: transformers.TransformWithStruct(&mixpanel.ExportEvent{}, client.SharedTransformers()...),
+		Columns: []schema.Column{
+			{
+				Name:     "project_id",
+				Type:     schema.TypeInt,
+				Resolver: client.ResolveProjectID,
+				CreationOptions: schema.ColumnCreationOptions{
+					PrimaryKey: true,
+				},
+			},
+			{
+				Name:     "time",
+				Type:     schema.TypeTimestamp,
+				Resolver: resolveExportTime,
+				CreationOptions: schema.ColumnCreationOptions{
+					PrimaryKey: true,
+				},
+			},
+		},
+	}
+}
+
+func fetchExportEvents(ctx context.Context, meta schema.ClientMeta, _ *schema.Resource, res chan<- any) error {
+	cl := meta.(*client.Client)
+
+	ret, err := cl.Services.ExportData(ctx, cl.MPSpec.StartDate, cl.MPSpec.EndDate)
+	if err != nil {
+		return err
+	}
+	res <- ret
+	return nil
+}
+
+func resolveExportTime(_ context.Context, meta schema.ClientMeta, r *schema.Resource, c schema.Column) error {
+	e := r.Item.(mixpanel.ExportEvent)
+	ts, ok := e.Properties["time"]
+	if !ok {
+		return errors.New("event does not have a time property")
+	}
+	tf, ok := ts.(float64) // json.Number translates to float64
+	if !ok {
+		return fmt.Errorf("event time property is not a float: %T", ts)
+	}
+	return r.Set(c.Name, time.Unix(int64(tf), 0))
```

Comment:
Yeah to my previous comment, looks like we're discarding any potential granularity of the timestamp here and only using unix seconds, which (if used as a PK) will probably cause events to be overwritten incorrectly? Not sure but if Mixpanel timestamps use floats as their PK then we can probably solve it by passing the microseconds in here as well, but we should be sure then that destination databases also store it at that granularity...

Reply:
Time is seconds. It's probably not even unique including `event`.

Reply:
Fixed in [`5e55b2e` (#6935)](https://github.com/cloudquery/cloudquery/pull/6935/commits/5e55b2e12611e093a7695bd69ca8f9df1477e987)